cmake/3rdParty/Platform/Mac/BuiltInPackages_mac.cmake

```diff
@@ -47,3 +46,10 @@ ly_associate_package(PACKAGE_NAME lz4-1.9.3-vcpkg-rev4-mac
 ly_associate_package(PACKAGE_NAME azslc-1.8.9-rev1-mac                              TARGETS azslc                       PACKAGE_HASH ccadb9bb90db18cfb31d416f1da2a9429266f22c7375fd0acf5aed299d68a7fb)
 ly_associate_package(PACKAGE_NAME SQLite-3.37.2-rev2-mac                            TARGETS SQLite                      PACKAGE_HASH b7d9abdb68045003e030e1a9a805db1aefa5e8fde6dccfbb4fab3a06249a41fc)
 ly_associate_package(PACKAGE_NAME AwsIotDeviceSdkCpp-1.15.2-rev2-mac                TARGETS AwsIotDeviceSdkCpp          PACKAGE_HASH 4854edb7b88fa6437b4e69e87d0ee111a25313ac2a2db5bb2f8b674ba0974f95)
+
+set(AZ_USE_PHYSX5 OFF CACHE BOOL "When ON PhysX Gem will use PhysX 5 SDK")
```

Comment:
nit/minor: Feel free to ignore but CMake has the `option` command (instead of set) to make this sort of thing simpler (see https://cmake.org/cmake/help/latest/command/option.html)

Reply:
nice, looks like `option` would work too! 
I couldn't find any single use of `option()` in our cmake files though, so probably best to keep `set` for consistency?

Reply:
Ah yeah you could be right, no problem 👍